docs/admin/namespaces/README.md

```diff
@@ -159,8 +159,10 @@ users:
 The next step is to define a context for the kubectl client to work in each namespace. The value of "cluster" and "user" fields are copied from the current context.
 
 ```console
-$ kubectl config set-context dev --namespace=development --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
-$ kubectl config set-context prod --namespace=production --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
+$ kubectl config set-context dev --namespace=development \
+--cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
```

Comment:
I don't think we should rush it today

On Mon, Jul 20, 2015 at 5:51 PM, Chao Xu <notifications@github.com> wrote:

> In docs/admin/namespaces/README.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11635#discussion_r35060961>
> :
>
> > @@ -159,8 +159,10 @@ users:
> >  The next step is to define a context for the kubectl client to work in each namespace. The value of "cluster" and "user" fields are copied from the current context.
> >
> >  ```console
> > -$ kubectl config set-context dev --namespace=development --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
> > -$ kubectl config set-context prod --namespace=production --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
> > +$ kubectl config set-context dev --namespace=development \
> > +--cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
>
> @thockin <https://github.com/thockin>, Do you think this commit is going
> in today? If not, I prefer adding the indention after I have an automated
> tool, instead of fixing them manually today.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11635/files#r35060961>
> .
>